src/Actions.ts

```diff
@@ -108,7 +108,7 @@ export class Actions {
         }
     }
     
-    private onDeleteSelected(payload, poll: Poll): void {
+    private async onDeleteSelected(payload, poll: Poll): void {
```

Comment:
These return types now should be Promise<Void>

Reply:
Done